tests/unit/processes/ingest/test_doab_process.py

```diff
@@ -61,53 +63,41 @@ def mockOAIQuery(self, mocker):
 
     def test_runProcess_daily(self, testProcess, mocker):
         mockImport = mocker.patch.object(DOABProcess, 'importOAIRecords')
-        mockSave = mocker.patch.object(DOABProcess, 'saveRecords')
-        mockCommit = mocker.patch.object(DOABProcess, 'commitChanges')
 
         testProcess.process = 'daily'
         testProcess.runProcess()
 
         mockImport.assert_called_once
-        mockSave.assert_called_once
-        mockCommit.assert_called_once
+        testProcess.record_buffer.flush.assert_called_once
```

Comment:
I assume the () is optional, but there should be consistency across files.

Reply:
I think it is but it's a good callout - I'll make those changes!